controllers/configmap_controller_test.go

```diff
@@ -13,6 +15,231 @@ import (
 	"github.com/openshift/windows-machine-config-operator/pkg/wiparser"
 )
 
+func TestReconcileWindowsServices(t *testing.T) {
```

Comment:
Go unit  test functions should have the naming convention of Test($FUNCTION_NAME)
So in this case it would be TestSetValues